install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -69,88 +69,69 @@ Because the Aggregator PV is relatively small, the least expensive performance g
-  # the default of 25 is sufficient for 95%+ of users. This should only be modified
-  # after consulting with Kubecost's support team
-  numDBCopyPartitions: 1
   logLevel: info
 
-  # env: has been removed to avoid unknown issues that would be caused by
-  # customizations that were required to run aggregator in previous versions
-  # extraEnv: can be used to add new environment variables to the aggregator pod
-
-  # the below settings should only be modified with support from Kubecost staff
+  # How much data to ingest from the federated store bucket, and how much data
+  # to keep in the DB before rolling the data off.
+  # 
+  # Note: If increasing this value to backfill historical data, it will take
+  # time to gradually ingest & process those historical ETL files. Consider
+  # also increasing the resources available to the aggregator as well as the
+  # dbConcurrentIngestionCount.
+  # 
+  # default: 91
+  etlDailyStoreDurationDays: 365
 
   # How many threads the read database is configured with (i.e. Kubecost API /
   # UI queries). If increasing this value, it is recommended to increase the
   # aggregator's memory requests & limits.
   # default: 1
   dbReadThreads: 1
+
   # How many threads the write database is configured with (i.e. ingestion of
   # new data from S3). If increasing this value, it is recommended to increase
   # the aggregator's memory requests & limits.
   # default: 1
   dbWriteThreads: 1
+
   # How many threads to use when ingesting Asset/Allocation/CloudCost data
   # from the federated store bucket. In most cases the default is sufficient,
   # but can be increased if trying to backfill historical data.
   # default: 1
   dbConcurrentIngestionCount: 1
-  # dbCopyFull: "true" can improve the time it takes to copy the write DB, 
-  # at the expense of additional memory usages.
-  dbCopyFull: "false"
+
   # Memory limit applied to read database connections.
   # default: 0Gi is no limit
   dbMemoryLimit: 0Gi
+
   # Memory limit applied to write database connections.
   # default: 0Gi is no limit
   dbWriteMemoryLimit: 0Gi
-  # How much data to ingest from the federated store bucket, and how much data
-  # to keep in the DB before rolling the data off.
-  # 
-  # Note: If increasing this value to backfill historical data, it will take
-  # time to gradually ingest & process those historical ETL files. Consider
-  # also increasing the resources available to the aggregator as well as the
-  # refresh & concurrency env vars.
-  # 
-  # default: 91
-  etlDayStoreDurationDays: 91
+
+  # If "true" can improve the time it takes to copy the write DB, at the expense
+  # of additional memory usage.
+  # default: "false"
+  dbCopyFull: "false"
+
+  # The number of partitions the datastore is split into for copying. The higher
+  # this number, the lower the RAM usage but the longer it takes for new data to
+  # show in the Kubecost UI.
+  # default: 1
+  numDBCopyPartitions: 1
```

Comment:
@mbolt35 Similarly, reading this seemed to indicate that increasing the number of partitions/chunks would lead to higher volume of smaller sized INSERTS. Probably better for memory but takes longer to execute? Let me know if you agree/disagree.

Reply:
From response above: 

> The "chunking" in this case is sort of hand-wavy, as the window copies run serially, and the memory pool for the database will only cleanup when the full copy is complete.